demos/ASP.Net Core/ITOps.ViewModelComposition.Gateway/ComposableRouteHandler.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Primitives;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace ITOps.ViewModelComposition.Gateway
+{
+    class ComposableRouteHandler
+    {
+        public static async Task HandleGetRequest(HttpContext context)
+        {
+            var result = await CompositionHandler.HandleGetRequest(context);
+            if (result.StatusCode == StatusCodes.Status200OK)
+            {
+                string json = JsonConvert.SerializeObject(result.ViewModel, GetSettings(context));
+                context.Response.ContentType = "application/json; charset=utf-8";
```

Comment:
no, in this sample we don't respect `Accept` and we have no support for response formatters. It's just an assumption for the sample.
Do you think we should?